Extensions/xkit_preferences.css

```diff
@@ -336,9 +381,9 @@
 	height: 40px;
 	position: absolute;
 	width: 100%;
-	background: rgb(246,245,246);
+	background: var(--xkit-secondary);
 	bottom: 0; left: 0;
-	box-shadow: inset 0px 1px 3px rgba(0,0,0,0.12);
+	box-shadow: inset 0px 1px 3px var(--xkit-22-shadow);
```

Comment:
shouldn't this be `--xkit-11-shadow` since the previous value was `rgba(0,0,0,0.12)`?